pkg/kubecost/asset.go

```diff
@@ -2534,6 +2532,7 @@ type AssetSet struct {
 	sync.RWMutex
 	aggregateBy []string
 	assets      map[string]Asset
+	FromSource  string
```

Comment:
Same thing about a comment on this field.

Reply:
Done